### DISCUSSION TITLE
operation aborted by callbackError when uploading to Clublog

### DISCUSSION THREAD

**Discussion by aussiegeek** [magicbug/Cloudlog#1176] [Q&A]
When I visit the URL to trigger the Clublog upload, I get a `operation aborted by callbackError`.

According to the debug page, everything is green, and searching around I seen https://github.com/magicbug/Cloudlog/issues/978, but in my case there isn't even a file created in the uploads directory.

My instance of Clouglog isn't accessible from the internet, and I'm wondering if this is part of the issue

**Comment by magicbug** [comment]
Can Cloudlog access the internet however? even if it's not possible to access it the other way. should only need to make an outgoing request.

Once we know that can debug it some more!

**Reply by aussiegeek** [reply]
Yep, no restrictions on outbound access

**Comment by AndreasK79** [comment]
@aussiegeek you might find some helpful information here: https://github.com/magicbug/Cloudlog/issues/978
